### PR TITLE
feat: Bump git-operator

### PR DESCRIPTION
**Pull request by mikolajb**
**What problem does this PR solve?**:
Bumping git-operator

**Which issue(s) does this PR fix?**:
https://jira.nutanix.com/browse/NCN-101693

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [ ] If the PR adds a version bump, ensure there is no breaking change in Licensing model (or NA).
- [ ] If a chart is changed or app configuration is significantly changed, the chart version is correctly incremented (so that apps are not automatically upgraded from a previous version of DKP).

### CODE REVIEW

File: services/git-operator/0.1.0/git-operator-manifests/all.yaml [hunk 1]
```diff
@@ -741,6 +934,205 @@ spec:
       serviceAccountName: git-operator-controller-manager
       terminationGracePeriodSeconds: 10
 ---
+apiVersion: apps/v1
```
Comment (takirala) [inline services/git-operator/0.1.0/git-operator-manifests/all.yaml:937]: i am curious - how were we creating Statefulset so far (like in dev.11) ?

**Review by takirala — APPROVED** [review]

**Comment by gracedo** [comment]
license validation is failing with `2024-08-01 22:11:34 ERR     err="failed to read list of images: failed to parse container image name "docker.io/mesosphere/gitwebserver:${VERSION:=dev}": invalid reference format"` 🤔

File: services/git-operator/0.1.0/git-operator-manifests/all.yaml [hunk 2]
```diff
@@ -741,6 +934,205 @@ spec:
...
+        - mountPath: /admin-creds
+          name: htpasswd-admin
+          readOnly: true
+        - mountPath: /etc/nginx/tls
+          name: tls
+          readOnly: true
+      - env:
+        - name: GIT_VOLUME_PATH
+          value: /volumes/git
+        image: docker.io/mesosphere/gitwebserver:${VERSION:=dev}
```
Comment (gracedo) [inline services/git-operator/0.1.0/git-operator-manifests/all.yaml:1009]: looks like this is the offending line causing image validation to fail 🤔

File: services/git-operator/0.1.0/git-operator-manifests/all.yaml (hunk 1 above)
Comment (mikolajb) [reply services/git-operator/0.1.0/git-operator-manifests/all.yaml:937]: They were embedded in the operator binary

**Review by gracedo — APPROVED** [review]

**Pull request merged**